File: learn-react/TweetApp/backend-spring-boot/src/main/java/com/prem/myresume/service/MyResumeService.java
package com.prem.myresume.service;

import com.prem.myresume.dto.MyResumeDto;
import java.util.List;

public interface MyResumeService {
    List<MyResumeDto> upsertMyResumes(List<MyResumeDto> dtoList);

    MyResumeDto getResumeByUniqueId(String uniqueId);
}

File: learn-react/TweetApp/backend/src/routes/MyResume.service.js
// MyResume.service.js

const MyResumeModel = require('./MyResume.model');

const getResumeByUniqueId = async (uniqueId) => {
  return await MyResumeModel.findOne({ uniqueName: uniqueId });
};

module.exports = {
  getResumeByUniqueId,
};
